gen/pipeline/pipeline.go

```diff
@@ -0,0 +1,141 @@
+package pipeline
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/buildkite/go-pipeline"
+	"github.com/buildkite/go-pipeline/ordered"
+	"github.com/spf13/cobra"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/yaml"
+)
+
+//nolint:stylecheck // SCREAMING_CASE for pseudo consts is acceptable.
+var (
+	// Known/Permitted environment variables pulled from AWS secret store.
+
+	GITHUB_API_TOKEN  = secretEnvVar{SecretID: "sdlc/prod/buildkite/github_api_token"}
+	REDPANDA_LICENSE  = secretEnvVar{SecretID: "sdlc/prod/buildkite/redpanda_sample_license"}
+	REDPANDA_LICENSE2 = secretEnvVar{SecretID: "sdlc/prod/buildkite/redpanda_second_sample_license"}
+	SLACK_VBOT_TOKEN  = secretEnvVar{SecretID: "sdlc/prod/buildkite/slack_vbot_token"}
+
+	// Known/Permitted Agent Pools
+
+	AgentsPipeLineUploader = map[string]any{"queue": "pipeline-uploader"}
+	AgentsLarge            = map[string]any{"queue": "v6-amd64-builders-m6id"}
+)
+
+var suites = []TestSuite{
+	{
+		Name:     "unit",
+		Required: true,
+		Timeout:  15 * time.Minute,
+	},
+	{
+		Name:     "integration",
+		Required: false,
+		Timeout:  time.Hour,
+	},
+	{
+		Name:     "acceptance",
+		Required: false,
+		Timeout:  time.Hour,
+	},
+	{
+		Name:         "kuttl-v1",
+		Required:     false,
+		Timeout:      time.Hour,
```

Comment:
I checked and `kuttl-v1` test suite finishes within 1 hour and 2 minutes.
```suggestion
		Timeout:      (10 * time.Minute) + time.Hour,
```

![Screenshot 2025-01-30 at 13 17 40](https://github.com/user-attachments/assets/b3416a66-dcd2-4e62-af80-c7363f4db565)

Reply:
Updated to 90min